src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -758,6 +764,55 @@ export class ResultsViewPageStore extends AnalysisStore
         }
     }
 
+    /**
+     * Genes of 'single gene' queries
+     * (i.e. non-fusion queries, without an upstream or downstream gene)
+     */
+    @computed get singleHugoGeneSymbols() {
+        if (
+            this.urlWrapper.query.gene_list &&
```

Comment:
You can now use optional chaining to write this condition ```this.urlWrapper.query?.gene_list.length```
it will evaluate falsy if gene_list is undefined

Reply:
Fix at all three places that his was done.

Reply:
still seems not done?

Reply:
Yes it is done.